ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -18,30 +18,48 @@ function (
         }
     }
 
+    function trackLiveBlogEpic() {
+        // if there is already a data-tracked attribute than we don't need to set up tracking again
+        var liveBlogEpicContainer = document.querySelectorAll('.contributions-epic:not([data-tracked])');
```

Comment:
document.querySelectorAll will return a node list, so we should rename liveBlogEpicContainer to liveBlogEpicContainers as it may have multiple elements in it.